UserManual.md

```diff
@@ -119,7 +119,16 @@ tar xzf hello-2.10.tar.gz
 cd hello-2.10
 ./configure
 make
+
+# save permanently to the job's run archive directory
+mv ./hello "$ARCHIVE/"
+```
+
+If everything went well, the absolute path to the freshly compiled `hello` binary is `$LAMINAR_HOME/archive/hello/latest/hello`, which prints
 ```
+Hello, world!
+```
+upon execution.
```

Comment:
For me it definitely would have been. On my first try, I was like "Okay, the job ran and might have compiled something, but what and where is it?" I ended up reading what GNU Hello actually is and compiled it myself in a separate folder (after installing dependencies).

IMHO running the self-compiled binary printing `Hello, world!` is a perfect fit for a hello-world-example - otherwise, after working yourself through the docs, installing `laminar` from source, setting up system user and permissions and getting the web server running, you end up with ... nothing?

Although archives are covered in detail later, having the binary at hand let me feel like I *finally* did a real thing and made it clear to me where the journey goes. And I think other newcomers would feel the same. 

So I think it is absolutely relevant. Like a little bit of foreshadowing to convey the basic idea and an overall picture of `laminar`.

Reply:
Fair argument, I'll accept it.

Just a minor nit, suggest maybe dropping `absolute`, since `latest` is a symlink